lib/Runtime/Library/ArgumentsObject.h

```diff
@@ -130,7 +130,7 @@ namespace Js
     //   - When SetWritable/Enumerable/Configurable/Accessors is called, convert HeapArgumentsObject  to ES5HeapArgumentsObject by swapping vtable.
     //   - Override GetItemAt/SetItemAt -- for disconnected items force GetItem/SetItem to use "this" rather than from frameObject.
     //   - Override GetEnumerator -- take care of enumerable = false arguments.
-    //   - The �disconnect� happens when (as ES5 spec says) setting writable to false and setting accessors.
+    //   - The "Disconnect" happens when (as ES5 spec says) setting writable to false and setting accessors.
```

Comment:
Can you make it "disconnect" instead of "Disconnect"? I believe capitalizing the D makes the meaning less clear.